moshi/src/main/java/com/squareup/moshi/-MoshiKotlinExtensions.kt

```diff
@@ -25,16 +27,19 @@ import kotlin.reflect.typeOf
  * @return a [JsonAdapter] for [T], creating it if necessary. Note that while nullability of [T]
  *         itself is handled, nested types (such as in generics) are not resolved.
  */
+@Deprecated("Use the Moshi instance version instead", level = DeprecationLevel.HIDDEN)
```

Comment:
is there a refactoring helper that can be added here, or is that infeasible for extension-to-instance?

Reply:
It's hidden and code simply needs to just be recompiled for this to work now, no refactoring needed!